codegen/scala-gen/src/main/scala/com/akkaserverless/codegen/scalasdk/impl/Types.scala

```diff
@@ -1,74 +0,0 @@
-/*
```

Comment:
This file changed too much, but still exists: https://github.com/lightbend/akkaserverless-java-sdk/pull/857/files#diff-0c057024dc21c735eec15f1d48e8051f473cbbf76c986a5ac0e0ba27477f07ac

Reply:
Yep, `git blame` seems to understand the move (though most of the lines still have changed...).